ledger-api/grpc-definitions/com/daml/ledger/api/v1/transfer.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+import "com/daml/ledger/api/v1/event.proto";
+import "google/protobuf/timestamp.proto";
+
+option java_outer_classname = "TransactionOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+// Complete view of an on-ledger transfer.
+message Transfer {
+
+  // Assigned by the server. Useful for correlating logs.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Required
+  string transaction_id = 1;
+
+  // TODO this field (whether it is populated) can signal whether it is originates from the same participant or not (
+  //  so additional retry etc procedures are in order or not), on the other hand with multi-hosted parties this is not
+  //  so obvious.
+  // The ID of the command which resulted in this transaction. Missing for everyone except the submitting party.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Optional
+  string command_id = 2;
+
+  // The workflow ID used in command submission. Only set if the ``workflow_id`` for the command was set.
+  // Must be a valid LedgerString (as described in ``value.proto``).
+  // Optional
+  string workflow_id = 3;
+
+  // Ledger effective time.
+  // Required
+  google.protobuf.Timestamp effective_at = 4;
+
+  // The absolute offset. The format of this field is described in ``ledger_offset.proto``.
+  // Required
+  string offset = 5;
+
+  oneof event {
+    TransferredOutEvent transfer_out_event = 6;
+    TransferredInEvent transfer_in_event = 7;
+  }
+}
+
+// Records that a contract has been transferred-out, and it becomes unusable on the source domain
+message TransferredOutEvent {
```

Comment:
Analogous to "create event" rather than "created event".
```suggestion
message TransferOutEvent {
```

Reply:
You caught me here. I didn't check how we've called `CreatedEvent` in LAPI 2.0. I just notice that we informally always speak about "create events" and not "created events".

Reply:
Usually, I have this habit that events should be past tense as opposed to the commands.